### ISSUE TITLE
Error when trying to run `gren make` on 0.5.0, only resolved after building from source

### ISSUE THREAD

**Issue by joeybright** [gren-lang/compiler#273]
I've had an odd series of events happen when trying to get 0.5.0 running on my machine.

First, I installed Gren 0.5.0 with `npm install -g gren-lang`. Then, I got the following error when running `gren make src/Main.gren`:

```dyld[12095]: Library not loaded: /nix/store/iz66rbm4c48m920zjiic3sc8ghl8pvgs-ncurses-6.4/lib/libncursesw.6.dylib
  Referenced from: <7C8E624D-C093-319B-9914-25CD81E031D1> /Users/joey/Library/Caches/gren/0.5.0/bin/gren
  Reason: tried: '/nix/store/iz66rbm4c48m920zjiic3sc8ghl8pvgs-ncurses-6.4/lib/libncursesw.6.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/nix/store/iz66rbm4c48m920zjiic3sc8ghl8pvgs-ncurses-6.4/lib/libncursesw.6.dylib' (no such file), '/nix/store/iz66rbm4c48m920zjiic3sc8ghl8pvgs-ncurses-6.4/lib/libncursesw.6.dylib' (no such file), '/usr/local/lib/libncursesw.6.dylib' (no such file), '/usr/lib/libncursesw.6.dylib' (no such file, not in dyld cache)
```

I tried the usual routes to fix the problem when I don't really have any clue what to do: Google and LLMs. Nothing I found and tried worked, and the error kept appearing when running `gren make`.

I then tried building the project from source. Following the README, I was able to get devbox to install all of the dependencies and build the project. The built executable worked! And now, oddly enough, running `gren make` (which, I assume, is running the originally installed gren executable from npm) works too!

While it's great 0.5.0 works for me now, I imagine others may run into this issue, so I thought I'd report it.

For some context, I'm on MacOS running Sequoia 15.2.

**Comment by robinheghan** [comment]
Maybe this happens _because of_ devbox/nix. I switched to devbox/nix not only for local development, but also for building the release builds of the compiler. It didn't occur to me that this might cause a runtime dependency on some nix-installed packages.

I'll try to confirm this theory, then cut a new release.

Thank you for reporting :)

**Comment by robinheghan** [comment]
Intel or Apple Silicon?

**Comment by robinheghan** [comment]
Fixed in 0.5.2
